src/ontology/edits/oeo-physical.omn

```diff
@@ -2792,6 +2792,117 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/356",
         <http://openenergy-platform.org/ontology/oeo/OEO_00000331>
     
     
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00010015>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Fossil hydrogen is hydrogen with fossil origin. It is usually produced from fossil fuels applying the steam reforming process.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/134
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/411",
+        rdfs:label "fossil hydrogen"@de
+    
+    SubClassOf: 
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000220>,
+        <http://openenergy-platform.org/ontology/oeo/has_origin> some <http://openenergy-platform.org/ontology/oeo/OEO_00030002>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00010016>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Synthetic hydrogen is hydrogen with synthetic origin. It is usually produced from water applying the water electrolysis process.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/134
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/411",
+        rdfs:label "synthetic hydrogen"@de
+    
+    SubClassOf: 
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000220>,
+        <http://openenergy-platform.org/ontology/oeo/has_origin> some <http://openenergy-platform.org/ontology/oeo/OEO_00030005>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00010017>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A synthetic fuel is a fuel produced from one or more portion(s) of matter using electric energy.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "e-fuel",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "electric fuel",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/134
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/411",
+        rdfs:label "synthetic fuel"@de
+    
+    EquivalentTo: 
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000173>
+         and (<http://openenergy-platform.org/ontology/oeo/has_origin> some <http://openenergy-platform.org/ontology/oeo/OEO_00030005>)
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/RO_0000091> some <http://openenergy-platform.org/ontology/oeo/OEO_00000097>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00010018>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Synthetic methane is synthetic fuel produced from water and carbon dioxide using electric energy applying the power-to-gas technology.",
```

Comment:
I don't know if I am too fussy, but for me power to gas is rather a concept than a technology. 
I would write: "Synthetic methane is a synthetic fuel produced from water and carbon dioxide using electric energy applying the electrolysis technology as first step and a methanation reaction as second step. This concept is known as power-to-gas."

Reply:
I would not include all the process details in the definition as have a `power-to-gas system` class in the ontology, too. There these process details would fit better. But we could use the `power-to-gas system` class in the definition. What about: _Synthetic methane is a synthetic fuel produced **in a power-to-gas system** from water and carbon dioxide using electric energy applying._?

Reply:
@stap-m : As you were involved in the issue discussion, what do you think?

Reply:
Sounds good to me